docs/internals/ARCHITECTURE.md

```diff
@@ -1,97 +1,92 @@
 # TigerBeetle Architecture
 
-This document is a short technical overview of the internals of TigerBeetle. It starts with the
-problem statement to motivate everything that follows, then gives the overview of TigerBeetle and
-provides motivation for a number of specific design decisions, and finally concludes with a list of
-prior art which inspired TigerBeetle.
+This document is a technical overview of the internals of TigerBeetle.  It includes a problem 
+statement, overview, motivation for design decisions and list of references that have inspired us.
 
 ## Problem Statement
 
 TigerBeetle is a database for workloads that:
 
-- are very contended and parallelize/shard poorly,
-- consist mostly of writes,
-- need very high throughput and moderately low latency,
-- require strong consistency guarantees,
-- and demand very high levels of durability.
+- are very contended and parallelize/shard poorly
+- consist mostly of writes
+- need very high throughput and moderately low latency
+- require strong consistency guarantees
+- demand very high levels of durability.
 
-At the moment, TigerBeetle is focused on financial transactions (transfers). Each transfer touches
-two accounts, and accounts are Pareto-distributed --- a few accounts are responsible for the bulk of
+At present, TigerBeetle is focused on financial transactions (transfers). Each transfer touches
+two accounts. Accounts are Pareto-distributed which means a few accounts are responsible for the bulk of
 the transfers. Multi-object transactions in combination with contention make traditional database
-architecture inefficient --- locking the two accounts and then going over the network to the
-application to compute the balances' update is slow, and, due to contention, this work can't be
+architecture inefficient. Locking the two accounts and then crossing the network to the
+application to compute the balances' update is slow. Due to contention, this work also can't be
 efficiently distributed across separate machines.
 
-While at the moment TigerBeetle implements only accounting logic, the underlying state machine can
-be swapped for a different one. TigerBeetle does only double-entry bookkeeping, but _can_ do
-anything that has the same combination of high performance and safety requirements, and high
-contention.
+TigerBeetle implements accounting logic but the underlying state machine can be swapped. TigerBeetle 
+does double-entry bookkeeping but _can_ do anything that has the same requirements.
```

Comment:
Let's keep the redundant repetition of "high performance, safety, extreme contention" --- while already mentioned above, it's worth circling around this more to make the key point sink.